### PR TITLE
Publish to cloudsmith

### PR DESCRIPTION
**Pull request by jayvdb**

### CODE REVIEW

File: butane_codegen/Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "butane_codegen"
-version = "0.5.0"
+version = "0.5.1-alpha.1"
```
Comment (joaommartins) [inline butane_codegen/Cargo.toml:3]:
```suggestion
version = "0.5.2-alpha.1"
```

**Review by joaommartins — CHANGES_REQUESTED** [review]

File: butane/Cargo.toml [hunk 2]
```diff
@@ -25,8 +25,8 @@ tls = ["butane_core/tls"]
 uuid = ["butane_core/uuid", "butane_codegen/uuid"]
 
 [dependencies]
-butane_codegen = { path = "../butane_codegen", version = "0.5" }
-butane_core = { path = "../butane_core", version = "0.5" }
+butane_codegen = { version = "0.5.1-alpha.1", registry = "franklin-ai-rosalind" }
```
Comment (joaommartins) [inline butane/Cargo.toml:28]:
```suggestion
butane_codegen = { version = "0.5.2-alpha.1", registry = "franklin-ai-rosalind" }
```

**Review by joaommartins — CHANGES_REQUESTED** [review]

File: butane_codegen/Cargo.toml (hunk 1 above)
Comment (jayvdb) [reply butane_codegen/Cargo.toml:3]: https://crates.io/crates/butane_codegen is a patch release behind the others

File: butane/Cargo.toml (hunk 2 above)
Comment (jayvdb) [reply butane/Cargo.toml:28]: see above

**Review by joaommartins — APPROVED** [review]

**Pull request merged**